docker-compose.yaml

```diff
@@ -4,7 +4,7 @@ volumes:
 services:
   db:
     platform: linux/x86_64
-    image: percona:5.7.35
+    image: percona:5.7.39
```

Comment:
@wadewinningham are you aware of this new image? Should be fine, right?

Reply:
I noticed this dependency's fallen behind what we typically use in nitro-web. I've captured an item to perform the upgrade here: https://github.com/powerhome/power-web-development-interview/issues/97